pages/mesosphere/dcos/1.12/release-notes/1.12.5/index.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes for 1.12.5
+title: Release Notes for 1.12.5
+menuWeight: 3
+excerpt: Release notes for DC/OS 1.12.5
+---
+
+DC/OS Version 1.12.5 was released on 13 January 2020.
+
+[button color="light" href="https://downloads.dcos.io/dcos/stable/1.12.5/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="purple" href="http://downloads.mesosphere.com/dcos-enterprise/stable/1.12.5/dcos_generate_config.ee.sh"]Download DC/OS Enterprise [/button]
+
+DC/OS 1.12.5 includes the following components:
+
+- Apache Mesos 1.7.3-dev [change log](https://github.com/apache/mesos/blob/d8acd9cfacd2edf8500f07f63a8837aa0ddd14ba/CHANGELOG)
+- Marathon 1.7.216 [change log](https://github.com/mesosphere/marathon/blob/9e2a9b579b968a2664df03099b03eaf86ffc7efc/changelog.md)
+- Metronome 0.6.33 [change log](https://github.com/dcos/metronome/blob/b8a73dd3cc3c2da035222031ccbbcf5c836ede7b/changelog.md)
+- Updated DC/OS UI to 1.12+v2.26.18.
+- Updated Signal service to release 1.6.0
+
+# Release summary
+
+DC/OS is a distributed operating system that enables you to manage resources, application deployment, data services, networking, and security in an on-premise, cloud, or hybrid cluster environment. 
+
+# Issues fixed in DC/OS 1.12.5
+The issues that have been fixed in DC/OS 1.12.5 are listed below. Most change descriptions include one or more issue tracking identifiers for reference.
+
+- The DC/OS diagnostics bundle now includes Mesos state logs content, located at `/var/log/mesos-state.tar.gz`. (DCOS-56403)
+- Added 3 new environment files to Mesos systemd. (DCOS-49092)
+- [Metronome] post-install configuration can be added to `/var/lib/dcos/metronome/environment` (DCOS_OSS-5509)
+- Changed `dcos-zk backup` and `dcos-zk restore` to exit early if ZooKeeper is running. (DCOS_OSS-5353)
+- Fixed preflight docker version check failing on unknown versions. (DCOS-56831)
+- DC/OS Net: Fix support for big sets in the IPset manager. (DCOS-57506, COPS-5229)
+- Added new diagnostic REST API bundle with performance improvements. (DCOS_OSS-5098)
+- Fixed old diagnostics bundle from increasing diagnostics job duration when job was done. (DCOS_OSS-5494)
+- Removed `nogroup` creation. (COPS-5220, DCOS-59247) 
```

Comment:
incorrect JIRA: DCOS-59427

Rephrased: "Updated DC/OS installation to not create `nobody` Linux user group on the hosts."

Reply:
Done